setup.py

```diff
@@ -121,6 +121,55 @@ def build_extensions(self):
 REQUIRES = [r.replace('[array]', '') for r in REQUIRES]
 
 
+compile_flags = ['-fopenmp']
```

Comment:
https://cython.readthedocs.io/en/latest/src/userguide/parallelism.html#compiling says:
```
For Microsoft Visual C++ compiler, use '/openmp' instead of '-fopenmp'.
```

Should we consider also Win?